src/bsoncxx/builder/core.hpp

```diff
@@ -263,6 +268,11 @@ class BSONCXX_API core {
     void append(const oid& value);
 
     ///
+    /// Append a decimal128 object as a BSON Decimal128.
+    ///
+    void append(const decimal128& value);
```

Comment:
That's close to true, but not quite true.  All of the append() overloads taking a b_xxx type pass a const reference (probably just to be consistent with each other), though the append() overloads taking small raw types pass by value:

```
240     ///
241     /// Append a native boolean as a BSON boolean.
242     ///
243     void append(bool value);
244
245     ///
246     /// Append a native double as a BSON double.
247     ///
248     void append(double value);
249
250     ///
251     /// Append a native int32_t as a BSON 32-bit signed integer.
252     ///
253     void append(std::int32_t value);
254
255     ///
256     /// Append a native int64_t as a BSON 64-bit signed integer.
257     ///
258     void append(std::int64_t value);
```